src/readsbml.jl

```diff
@@ -147,7 +147,7 @@ end)
 function readSBML(
     fn::String,
     sbml_conversion = document -> nothing;
-    report_severities = ["Fatal", "Error"],
+    report_severities = ["Fatal", "Error"]
```

Comment:
Why removing the commas?  They're useful to add a new argument without disrupting git history, which is exactly what you're doing here.

Reply:
i think the vscode auto formatter did that

Reply:
JuliaFormatter to the git hook! :D